src/Inputs/scene.inputManager.ts

```diff
@@ -267,7 +267,10 @@ export class InputManager {
      * @param pointerEventInit pointer event state to be used when simulating the pointer event (eg. pointer id for multitouch)
      */
     public simulatePointerMove(pickResult: PickingInfo, pointerEventInit?: PointerEventInit): void {
-        let evt = new PointerEvent("pointermove", pointerEventInit);
+        let evt: any = new PointerEvent("pointermove", pointerEventInit);

```

Comment:
why `any`? same below

Reply:
Removed any, added deviceType, deviceSlot, and inputIndex to UIEvent declaration in browser.d.ts